series-short.bib

```diff
@@ -7,6 +7,7 @@ @comment{The
 @comment{Here are some characters that might be useful, so you can copy and paste them:}
 @comment{é ß ó ñ á ü Á ë ú ó Ş ö ä ã ç č ø ł è ô}
 
+@string{ srs_AISC     = "AAMAS" }
```

Comment:
srs_AISC already exists (see two lines down). If the key should be srs_AAMAS, please also include in series.bib